ccd-definition/CaseEvent/User/UserEvents-WelshEnhancements-nonprod.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
```

Comment:
"EventEnablingCondition" can be added for this event to make it visible only for the Lip cases.

Reply:
Done